src/transaction/log_replication_atomic.hpp

```diff
@@ -90,6 +90,9 @@ namespace cublog
       void set_lowest_unapplied_lsa ();
       void replicate_sysop_start_postpone (cubthread::entry &thread_entry, const LOG_RECORD_HEADER &rec_header);
 
+      void bookkeep_classname (cubthread::entry &thread_entry, const OID *classoid);
```

Comment:
I would move all DDL related code to a separate helper.
What do you think? Maybe in a separate PR?